domain/build.gradle

```
@@ -0,0 +1,22 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
```

```suggestion
plugins {
    id 'kotlin'
}
```

java-library는 빼셔도 무방합니다.